src/main/java/io/odpf/firehose/metrics/Instrumentation.java

```diff
@@ -199,6 +186,12 @@ public void captureGlobalMessageMetrics(Metrics.MessageScope scope, int counter)
         statsDReporter.captureCount(GLOBAL_MESSAGES_TOTAL, counter, String.format(MESSAGE_SCOPE_TAG, scope.name()));
     }
 
+    public void captureGlobalMessageMetrics(Metrics.MessageScope scope, int counter, String... tags) {
```

Comment:
we can edit the previous method rather than adding a new one.

Reply:
changed